Prox/Prox/Widgets/ReviewScoreView.swift

```diff
@@ -62,34 +57,58 @@ class ReviewScoreView: UIStackView {
         }
     }
 
+    private func updateReviewScore(score: Float) {
+        for (index, subview) in arrangedSubviews.enumerated() {
+            if let scoreItem = subview as? ReviewScoreItemView {
+                let scoreLeft = self.score - Float(index)
+                setFillScore(score: scoreLeft, forView: scoreItem)
+            }
+        }
+    }
+
+    private func setFillScore(score: Float, forView view: ReviewScoreItemView) {
```

Comment:
This could be a method on `ReviewScoreItemView` which (as per my comment below) might be better suited to take a `FillAmount` enum rather than a Float, but if it works now, it's probably fine.